app/controllers/application_controller.rb

```diff
@@ -39,116 +39,81 @@ def routing_error
-        if exception.generic_error?
-          log_message_to_sentry(exception.va900_warning, :warn, i18n_exception_hint: exception.va900_hint)
-        end
-      end
-    end
-
-    va_exception =
-      case exception
-      when Pundit::NotAuthorizedError
-        Common::Exceptions::Forbidden.new(detail: 'User does not have access to the requested resource')
-      when Common::Exceptions::TokenValidationError
-        Common::Exceptions::Unauthorized.new(detail: exception.detail)
-      when ActionController::ParameterMissing
-        Common::Exceptions::ParameterMissing.new(exception.param)
-      when ActionController::UnknownFormat
-        Common::Exceptions::UnknownFormat.new
-      when Common::Exceptions::BaseError
-        exception
-      when Breakers::OutageException
-        Common::Exceptions::ServiceOutage.new(exception.outage)
-      when Common::Client::Errors::ClientError
-        # SSLError, ConnectionFailed, SerializationError, etc
-        Common::Exceptions::ServiceOutage.new(nil, detail: 'Backend Service Outage')
-      else
-        Common::Exceptions::InternalServerError.new(exception)
-      end
-
-    unless skip_sentry_exception_types.include?(exception.class)
-      va_exception_info = { va_exception_errors: va_exception.errors.map(&:to_hash) }
-      log_exception_to_sentry(exception, extra.merge(va_exception_info))
-    end
-
-    headers['WWW-Authenticate'] = 'Token realm="Application"' if va_exception.is_a?(Common::Exceptions::Unauthorized)
-    render json: { errors: va_exception.errors }, status: va_exception.status_code
-  end
-  # rubocop:enable Metrics/BlockLength
-
-  def set_tags_and_extra_context
-    Thread.current['request_id'] = request.uuid
-    Thread.current['additional_request_attributes'] = {
-      'remote_ip' => request.remote_ip,
-      'user_agent' => request.user_agent
-    }
-    Raven.extra_context(request_uuid: request.uuid)
-    Raven.user_context(user_context) if current_user
-    Raven.tags_context(tags_context)
+  def set_app_info_headers
+    headers['X-Git-SHA']           = AppInfo::GIT_REVISION
+    headers['X-GitHub-Repository'] = AppInfo::GITHUB_REPO
   end
 
-  def user_context
-    {
-      uuid: current_user&.uuid,
-      authn_context: current_user&.authn_context,
-      loa: current_user&.loa,
-      mhv_icn: current_user&.mhv_icn
-    }
+  def handle_error(error)
+    Common::Client::ErrorHandler.handle(error)
   end
 
-  def tags_context
-    { controller_name: controller_name }.tap do |tags|
-      if current_user.present?
-        tags[:sign_in_method] = current_user.identity.sign_in[:service_name]
-        # account_type is filtered by sentry, becasue in other contexts it refers to a bank account type
-        tags[:sign_in_acct_type] = current_user.identity.sign_in[:account_type]
-      else
-        tags[:sign_in_method] = 'not-signed-in'
-      end
-    end
+  def render_error(exc)
+    render json: { errors: exc.errors }, status: exc.status
   end
```

Comment:
Signature here isn't great - use "exception" or something more descriptive than `exc` and perhaps add some comments